src/hooks/consts.ts

```diff
@@ -0,0 +1,4 @@
+export const RIGHT = 'right';
+export const LEFT = 'left';
+export const DOWN = 'down';
+export const UP = 'up';
```

Comment:
Now that we're in ts land, an enum is a much nicer way to bind these together.
Also, the file name `consts` doesn't make sense here anymore. You can move these directly to the `useKeyboardNavigation` hook file.

```suggestion
export enum KeyboardArrows {
  RIGHT = 'right',
  LEFT = 'left',
  DOWN = 'down',
  UP = 'up',
}
```